package.json

```diff
@@ -2,20 +2,27 @@
   "name": "ftlabs-email-signatures",
   "description": "A Chrome Extension that automagically adds links from RSS feeds to the end of emails sent in the Gmail web client",
   "scripts": {
-    "build-js": "mkdir -p build/scripts; rm -f build/scripts/*; find ./src/scripts -maxdepth 1 -name '*.js' -execdir browserify {} -t [ babelify  --sourceMapRelative . --presets [es2015] ] -o ../../build/scripts/{} ';'",
+    "build-js": "gulp build",
     "watch": "npm run build && sh -c 'nodemon -e js --watch src/scripts/ --exec \"npm run deploy\" & wait'",
```

Comment:
Could you add `verify` as an npm task please?

And document these in the readme? :-)